Week 9/Marvel/app/src/test/java/com/carol/musyoka/marvel/marvelapi/characters/CharactersViewModelTest.groovy

```diff
@@ -0,0 +1,21 @@
+package com.carol.musyoka.marvel.marvelapi.characters
```

Comment:
Nice job on including a viewModel test, it's been awhile since I've seen groovy!